src/helpers/fields.js

```diff
@@ -16,15 +16,11 @@ export function makeField(value, optionalDirty) {
  * Walks through an object hierarchy of fields and marks everything dirty.
  */
 export function dirtyAllFields(field) {
-  if (
-    _.keys(field).length === 2 &&
-    _.has(field, 'value') &&
-    _.has(field, 'dirty')
-  ) {
+  if (keys(field).length === 2 && has(field, 'value') && has(field, 'dirty')) {
```

Comment:
We can use the [native `Object.keys(field)`](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_keys). A native `has` isn't as simple.

```suggestion
  if (Object.keys(field).length === 2 && has(field, 'value') && has(field, 'dirty')) {
```